os/drivers/wireless/bluetooth/bt_null.c

```diff
@@ -242,6 +256,94 @@ static void btnull_process_write_local_name_req(FAR struct bt_buf_s *buf)
 	}
 }
 
+static void btnull_format_le_adv_report(FAR struct bt_buf_s *buf)
+{
+	struct bt_hci_evt_hdr_s evt;
+	struct bt_hci_evt_le_meta_event_s cmd;
+	struct bt_hci_ev_le_advertising_info_s adv_info = {0, };
+	FAR uint8_t *data = buf->frame->io_data;
+	int ndx;
+	int len;
+	uint8_t num_reports = 3;
+	uint8_t rssi = 0x9f;
+
+	nvdbg("format_le_adv_report entered.\n\n");
+
+	/* Minimal setup for the command complete event */
+
+	len = sizeof(struct bt_hci_evt_hdr_s)
+			+ sizeof(struct bt_hci_evt_le_meta_event_s)
+			+ (num_reports * (sizeof(struct bt_hci_ev_le_advertising_info_s) + sizeof(uint8_t) + 31 + 1));
```

Comment:
I think, there is problem in len, can you please modify as below       
 len = sizeof(struct bt_hci_evt_hdr_s)
                        + sizeof(struct bt_hci_evt_le_meta_event_s)
                        + sizeof(uint8_t)
                        + (num_reports * (sizeof(struct bt_hci_ev_le_advertising_info_s) + 31 + 1));

Reply:
You are right. Thank you.